config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -72,6 +72,33 @@ presubmits:
         - --warnings=unknown-fields-all
     annotations:
       testgrid-dashboards: presubmits-test-infra
+  - name: pull-test-infra-prow-checkconfig-loose
+    optional: true
+    decorate: true
+    skip_report: true
+    run_if_changed: '^(config/prow/(config|plugins).yaml$|config/jobs/.*.yaml$)'
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/checkconfig:v20230124-17d9ffa086
+        command:
+        - checkconfig
+        args:
+        - --config-path=config/prow/config.yaml
+        - --job-config-path=config/jobs
+        - --plugin-config=config/prow/plugins.yaml
+        - --warnings=mismatched-tide-lenient
+        - --warnings=tide-strict-branch
+        - --warnings=needs-ok-to-test
+        - --warnings=validate-owners
+        - --warnings=missing-trigger
+        - --warnings=validate-urls
+        - --warnings=unknown-fields
+        - --warnings=duplicate-job-refs
+        - --warnings=unknown-fields-all
+        - --warnings=non-decorated-jobs
+        - --warnings valid-decoration-config
```

Comment:
```suggestion
        - --warnings=valid-decoration-config
```

Reply:
yeah, the space won't work here because this is a single argv entry